controllers/task.js

```diff
@@ -5,6 +5,7 @@ const { sendResponse } = require('../helpers/sendResponse');
 const { populate } = require('../models/ratings');
```

Comment:
This is wrong no populate function is present in models/ratings and line is having no use please remove this